Program.cs

```diff
@@ -1,4 +1,23 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
-
-   
\ No newline at end of file
+if (int.TryParse (args[0], out int num)) Console.WriteLine ($"The {num}th Armstrong number is: " + NthArmstrongNumber (num));
+else  Console.WriteLine ("Invalid Input");
+static int NthArmstrongNumber (int num) {
+   int count = 0, i = 0;
+   if (num < 10) return num;
+   else {

```

Comment:
You can remove the else part here. For number less than 10 you have already handled in the above line.
Anyway, the below logics need to be executed for any other numbers. So else part is not required.
There is nothing wrong with the implementation (else part) you applied but removing that would make more sense and provide cleaner code.

Reply:
just else condition alone.